srcpkgs/perl-autovivification/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'perl-autovivification'
+pkgname=perl-autovivification
+version=0.18
+revision=1
+wrksrc="${pkgname/perl-/}-${version}"
+build_style=perl-module
+hostmakedepends="perl"
+makedepends="${hostmakedepends}"
+depends="${makedepends}"
```

Comment:
In cases like this maybe just
```
hostmakedepends="perl"
makedepends="perl"
depends="perl"
```
would be enough, because it’s shorter and better readable? But it’s not wrong and I guess just a matter of taste. ;)